Source/Scene/Instanced3DModel3DTileContent.js

```diff
@@ -419,8 +419,9 @@ define([
             instanceScale.x = 1.0;
             instanceScale.y = 1.0;
             instanceScale.z = 1.0;
-            var scale = featureTable.getProperty('SCALE', i, ComponentDatatype.FLOAT);
+            var scale = featureTable.getProperty('SCALE', i, ComponentDatatype.FLOAT, 1);
```

Comment:
Oh... yes that's right. Thanks for the refresher.

So yes, the code you suggested above does sounds correct.